src/css/elements.css

```diff
@@ -196,7 +196,7 @@ div.blockquote {
 
 
 .hovertip {
-	width: 15% !important;
+	width: 25% !important;
```

Comment:
Maybe make it a bit wider? On some screen sizes this could still be too small

Reply:
On mobile the width is increased to 90%, though because the page content width can vary, I'd prefer a static size for desktop - maybe something like `min(90%, 15rem)` (the page content width is 44.25rem) and then ditch the media query

Reply:
Okay that's fair